.circleci/config.yml

```diff
@@ -92,7 +91,10 @@ jobs:
           path: test-results/unit-test-reports.html
 
   integration_test:
-    executor: integration-tests
+    executor:
+      name: hmpps/node_redis
+      node_tag: << pipeline.parameters.node-version >>
+      redis_tag: buster
```

Comment:
think this should probably be `4-buster` to match the elasticache image, but was already set incorrectly so :shrug: